fuzzinator/listener/listener_manager.py

```diff
@@ -12,6 +12,19 @@
 
 logger = logging.getLogger(__name__)
 
+class _Trampoline(object):
```

Comment:
I think the code base doesn't use underscore-prefixed class names, not even for module-private entities. Such 'private' entities are simply not published by the containing package in `__init__.py:__all__`. (It can be questioned whether this is pythonic or not, but this is current practice.) In a different (but related) project, a similar issue had to be handled already, also because of Windows support, see https://github.com/renatahodovan/picire/commit/f7cc221b6be094842e981cde3cfd3103922bfe9a#diff-9dceab1274f9deef93289e7d2e1a84e9 . There, internal classes got moved to global scope, too, sometimes prefixed by some meaningful name. This approach should be followed here, too. So, the class should either remain `Trampoline` or become `ListenerManagerTrampoline`.

Reply:
I'm okay with that. I've just followed the good old way to indicate that it is "private". Will update the PR to use `Trampoline` name.